subsys/shell/backends/Kconfig.backends

```diff
@@ -206,6 +206,14 @@ config SHELL_BACKEND_RTT_BUFFER
 	  Select index of up-buffer used for shell output, by default it uses
 	  terminal up-buffer and its settings.
 
+config SHELL_RTT_INACTIVITY_PERIOD
+	int "Inactivity period implying host disconnection"
```

Comment:
I think we need a unit of measurement here.

Reply:
done